test/test_pipeline/test_losses.py

```diff
@@ -37,6 +37,7 @@ def test_get_name_error():
 
 @pytest.mark.parametrize('weighted', [True, False])
 def test_losses(weighted):
+    torch.manual_seed(1)
```

Comment:
why did you have to seed the training? is m wondering if we should do this (if needed) in conftest

Reply:
basically to determine the weights for each class the labels generated should contain all 4 classes. But because its random sometimes the labels array does not have a class and this causes weights to be inferred for only three classes and gives an error when we try to use them in cross entropy